README.md

```diff
@@ -2,39 +2,45 @@
 This utility patches the official AM2R 1.1 release (Windows) to the fan-made Community Update (Linux).
 
 ## Dependencies
-The patcher and the installer only require a small amount of dependencies:
+The patcher and the installer only require a small amount of dependencies. Note that the Java-related packages (`java`, `jre`, `jdk`) packages are only required for building an Android APK.
 
 ### Arch (including Manjaro, EndeavourOS, RebornOS, etc.)
-Make sure that multilib is enabled, as `lib32-libpulse` is a 32-bit library, and Arch does not enable 32-bit support by default.  
+Make sure that multilib is enabled, as `lib32-libpulse` is a 32-bit library, and Arch does not enable 32-bit support by default.
 To enable it, go to `/etc/pacman.conf`, search for `[multilib]`, and make sure that the next two lines are uncommented:
 ```
 [multilib]
 Include = /etc/pacman.d/mirrorlist
 ```
 
 Then install the following dependencies:  
-`sudo pacman -S --needed unzip sed xdelta3 jre8-openjdk lib32-libpulse`
+`sudo pacman -S --needed unzip sed patchelf xdelta3 jre8-openjdk lib32-libpulse`
 
 
 ### Debian (including Ubuntu, Mint, PopOS, etc.)
-Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.  
+Make sure that the i386 architecture is enabled for you, as libopenal1 requires the 32-bit version, and Ubuntu does not enable 32-bit support by default.
```

Comment:
Should be "Make sure you enable the i386 architecture, ..."

Remove comma after 'version'.